### ISSUE TITLE
Incorrect MDTextArea behavior in nested lists

### ISSUE THREAD

**Issue by Cvolton** [geode-sdk/geode#237]
### Platform

Windows

### Geode Version

v1.3.1

### Mods Installed

* absolllute.megahack v7.1.1-beta.5
* camila314.betterloading v1.0.2-beta
* cattodev.gatobot v1.3.0
* cvolton.betterinfo v3.0.4 (dev build with the respective broken markdown included)
* cvolton.capacity_fix v1.0.1
* cvolton.robtop_bot v1.0.1
* geode.devtools v1.2.1
* hjfod.gdshare v1.0.1
* hjfod.gmd-api v1.0.2
* mat.practice-music v1.0.5

### Expected Behavior

Expected the MD to render as follows:

* Extended Level Info:
  * Added exact upload dates
  * Added caching of relative upload dates
  * Improved loading performance
* View Leaderboard As:
  * Enabled on Mac
  * Added loading circle

### Actual Behavior

![2023-09-12_17-41_geometrydash_Z4RE2sW1PT](https://github.com/geode-sdk/geode/assets/5721187/a513c1cb-c6aa-4dda-8aee-e97ed9fa31e7)

### Steps to Reproduce

```md
* Extended Level Info:
  * Added exact upload dates
  * Added caching of relative upload dates
  * Improved loading performance
* View Leaderboard As:
  * Enabled on Mac
  * Added loading circle
```

### Additional Information

While my testing applies to v1.3.1, MDTextArea doesn't appear to have been touched since then in any branches

**Comment by altalk23** [comment]
This is fixed in 1.4.0 but i think i should move it to the main too

**Comment by altalk23** [comment]
Fixed by f64c74a
